### DISCUSSION TITLE
Spells being Proc'd for NPCs attack/defense.

### DISCUSSION THREAD

**Discussion by HMxMisfit** [AscensionGameDev/Intersect-Engine#2095] [Ideas]
When a NPC attacks or defends it has a % chance to cast a spell.

The proc would happen when health damage is received or given.

**Comment by PyroTech03** [comment]
So NPC's version of the equipment one?

Otherwise couldn't you just give them the spell?

**Reply by HMxMisfit** [reply]
Yes it would be useful for more unique enemies. It would be more reactive and less about giving them a spell to cycle through.
